### ISSUE TITLE
Upgrade or replace CKEditor4

### ISSUE THREAD

**Issue by johnatawnclementawn** [archesproject/arches#9738]
CKEditor4 has reached [end of life](https://ckeditor.com/blog/ckeditor-4-end-of-life/). We should look at upgrading or replacing this component.

**Comment by chiatt** [comment]
This is likely blocked by an upgrade to bootstrap 5

**Comment by johnatawnclementawn** [comment]
Closing in favor of #10828
